### DISCUSSION TITLE
Achievements

### DISCUSSION THREAD

**Discussion by BreezeWhite** [Music-and-Culture-Technology-Lab/omnizart#56] [Show and tell]
Ranked 1st research on [paperswithcode.com](https://paperswithcode.com/) trending (2021/12/23).

![Screenshot from 2021-12-23 19-58-47](https://user-images.githubusercontent.com/24308057/147237610-b3380d7c-4e50-4e4f-8b01-d20c9aba45a6.png)

**Comment by BreezeWhite** [comment]
Ranked 3th hot news on [Hacker News](https://news.ycombinator.com/) (2021/12/19).

![image](https://user-images.githubusercontent.com/24308057/147238200-346a9f89-ade5-48a7-9e31-d8bf18edcb2e.jpeg)

**Comment by BreezeWhite** [comment]
**1k stars achieved!!**

![image](https://user-images.githubusercontent.com/24308057/147324490-403436e4-7059-489a-8c22-0dffd1c739d3.jpeg)

**Reply by yoyolicoris** [reply]
![image](https://user-images.githubusercontent.com/17811395/147325859-b275c9d2-6bfa-4e73-8210-f95d50407e48.png)
